### ISSUE TITLE
GVExport is not working if the custom theme "modern" is activated

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#202]
see [issue](https://github.com/JesseWebDotCom/webtrees-theme-modern/issues/2)

**Comment by Neriderc** [comment]
I tried to test it but I can't get the theme to work at all. Once installed, trying to switch to the modern theme gives me this error:
````
ErrorException: readfile(http://localhost:8089/module/_webtrees-theme-modern_/Asset?asset=css%2Fpalettes%2Fcolorful%20%28white-blue-pink%29-min.css&amp;hash=1663011512): Failed to open stream: Cannot assign requested address in /var/www/webtrees/modules_v4/webtrees-theme-modern/resources/views/layouts/default.phtml:61
````

That's on my localhost development version. I tried on my production instance and the theme doesn't properly load. Most of the page has no styling applied, and I get a big black circle at the bottom of each page. This is not even looking at GVExport, it's like this on the home page.

I noticed in that repo that you had also had issues with errors. Was there anything special you needed to do to get it to work?

**Comment by Neriderc** [comment]
Hmm it might be easiest to wait for them to respond to the other issue you created.

My guess is that their theme causes a javascript error of some sort, which may prevent other javascript running on the page, which would prevent GVExport from working. You could have a look at the browser console and see if there is  a javascript error.

**Comment by Neriderc** [comment]
I'm still not able to get this installed.

Did you want to try troubleshooting on your end? (probably by looking at the JavaScript console to see which module is causing an error)

Or should we close this for now and revisit at a later date when the theme is more production ready?

**Comment by hartenthaler** [comment]
close it for now

**Comment by Neriderc** [comment]
I stumbled across the modern theme today, and remembered this issue. The latest version installs ok for me, so I installed it and found the issue. GVE is trying to measure the distance between the header and footer to set the size, but this theme seems to not have the original footer element (class doesn't exist) and instead inserts it's own.

I've set a fallback for if the footer isn't found. It won't nicely measure the distance but it will load the diagram, and should have the height in the right ballpark.

It's working for me now, hopefully it works for you too!
